compliance/azure/instances_without_fnm_agent/README.md

```diff
@@ -0,0 +1,92 @@
+## Check for Azure instances not running FNMS agent
+
+### What it does
+This policy checks all instances running in Azure to determine if the FlexNet Inventory Agent is running on the instance and reports on any that are missing the agent.
+The policy is a report only policy, no action is taken during the Policy Escalation.
```

Comment:
replace report with recommendation.  Product want's to avoid the word report for policies.

Reply:
Made the required changes. Thanks.

Reply:
Do I need to replace "report"  with "recommendation" in all places or only for line number 5?